ldb_testing.go

```diff
@@ -162,6 +162,29 @@ func (tu *LDBTestUtil) InsertRows(family string, table string, rows [][]interfac
 	}
 }
 
+// DeleteAll deletes all rows from the given table.
+func (tu *LDBTestUtil) DeleteAll(family string, table string) {
+	hunks := []string{
+		"DELETE FROM",
+		fmt.Sprintf("%s___%s", family, table),
+	}
+
+	qs := strings.Join(hunks, " ")
+	_, err := tu.DB.Exec(qs)
+	if err != nil {
+		tu.T.Fatalf("Unexpected error deleting data: %+v", err)
+	}
+
+	qs = fmt.Sprintf(
```

Comment:
I'm not sure we need to update the last ledger update table here..  Updating that table was added to `InsertRows` in order to to simulate what the reflector does when pulling updates from the ledger.

Reply:
Got it. Thanks for the feedback. Will remove it.